### ISSUE TITLE
[Feature Request]: Camelise component parameters and handle snake->camel on incoming data and camel->snake on outgoing data

### ISSUE THREAD

**Issue by MbBrainz** [fabien0102/openapi-codegen#213]
According to the Google Typescript style guide, we should use[ lowerCamelCase for parameters](https://google.github.io/styleguide/tsguide.html#naming-rules-by-identifier-type). 

Currently, because the openAPI standard is with snake case the current codegen generates types with snake_case parameters. This makes sense, because you wouldnt need a conversion step here, however it would be nice to have the option to generate lowerCamelCase parameter names. 

personal context: Im working on a library where they are already using lowerCamelCase and manually performing all the conversions, but they still manually update that library when the api changes. This is imo a waste of engineering time, but they cant just drop the CamelCase, cause there are many users of their library already.

**Comment by MbBrainz** [comment]
Your argument is completely fair @fabien0102 thanks for the elaborate perspective. Im still sad about it though :(

**Comment by fabien0102** [comment]
Yeah, it’s always a pain point when you have different styles between frontend/backend/services… Definitely something that could be cool to solve, but let’s be honest, for now, I don’t have the bandwidth for it and half solving this will be worse than not solving it 😓
